File: VB/ChartLesson3/Models/GSP.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.Data.Entity.Spatial

<Table("GSP")>
Partial Public Class GSP
    Public Property ID As Integer

    <StringLength(50)>
    Public Property Region As String

    Public Property Year As Integer?

    <Column("GSP")>
    Public Property GSP1 As Double?

    <Column(TypeName:="timestamp")>
    <MaxLength(8)>
    <Timestamp>
    Public Property SSMA_TimeStamp As Byte()
End Class

File: CS/ChartLesson3/Models/GSP.cs
namespace ChartLesson3.Models
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("GSP")]
    public partial class GSP
    {
        public int ID { get; set; }

        [StringLength(50)]
        public string Region { get; set; }

        public int? Year { get; set; }

        [Column("GSP")]
        public double? GSP1 { get; set; }

        [Column(TypeName = "timestamp")]
        [MaxLength(8)]
        [Timestamp]
        public byte[] SSMA_TimeStamp { get; set; }
    }
}
